installer/Config.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Installer;
+
+class Config
+{
+	private $config
+		= [
+			/*
+			|--------------------------------------------------------------------------
+			| Server Requirements
+			|--------------------------------------------------------------------------
+			|
+			| This is the default Laravel server requirements, you can add as many
+			| as your application require, we check if the extension is enabled
+			| by looping through the array and run "extension_loaded" on it.
+			|
+			*/
+			'core'         => ['minPhpVersion' => '7.2.0',],
+			'requirements' => [
+				'php'    => [
+					'openssl',
+					'pdo',
+					'mbstring',
+					'tokenizer',
+					'JSON',
+					'exif',
+					'gd',
+				],
+				'apache' => ['mod_rewrite',],
+			],
+			/*
+			|--------------------------------------------------------------------------
+			| Folders Permissions
+			|--------------------------------------------------------------------------
+			|
+			| This is the default Laravel folders permissions, if your application
+			| requires more permissions just add them to the array list bellow.
+			|
+			*/
+			'permissions'  => [
```

Comment:
I don't think that enforcing particular values of the permission bits is the right approach. For instance, what if I simply change the ownership of the whole Lychee tree to `www-data`? I don't need to set any group permissions then. Or, in my particular case, the directories are normally owned by `kamil:users` and changing them to `775` won't give the http daemon the permissions it needs but the installer will be happy. In `app/Http/Controllers/DiagnosticsController.php` we use `is_writeable()` which strikes me as much more robust (I am naively assuming that it's implemented sanely behind the scenes :smiley:).